src/main/java/teammates/storage/api/FeedbackResponsesDb.java

```diff
@@ -738,7 +738,9 @@ private FeedbackResponse getFeedbackResponseEntity(
        
         Query q = getPm().newQuery(FeedbackResponse.class);
         q.declareParameters("String feedbackQuestionIdParam, String giverSectionParam, String receiverSectionParam");
-        q.setFilter("feedbackQuestionId == feedbackQuestionIdParam && giverSection == giverSectionParam && receiverSection == receiverSectionParam");
+        q.setFilter("feedbackQuestionId == feedbackQuestionIdParam "
+                    + "&& giverSection == giverSectionParam "
+                    + "&& receiverSection == receiverSectionParam");
```

Comment:
Might have to consider if we need a separate way to write strings representing boolean conditions (like what was done for HTML strings), but I think this is fine as is :)